<launch>
  <node pkg="move_arm" type="move_arm_simple_action" output="screen"
  name="move_right_arm">    

    <remap from="robot_description" to="robot_description" />
    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />
    <remap from="joint_state" to="joint_states" />

    <remap from="arm_ik" to="pr2_right_arm_kinematics/get_collision_free_ik" />	
    <remap from="arm_fk" to="pr2_right_arm_kinematics/get_fk" />	

    <remap from="get_valid_state" to="ompl_search/find_valid_state" />

   <remap from="controller_start"  to="/r_arm_joint_waypoint_controller/TrajectoryStart" />
    <remap from="controller_query"  to="/r_arm_joint_waypoint_controller/TrajectoryQuery" />
    <remap from="controller_cancel" to="/r_arm_joint_waypoint_controller/TrajectoryCancel" />

    <remap from="filter_trajectory" to="trajectory_filter/filter_trajectory" />	
    
    <param name="group" type="string" value="right_arm" />
    <param name="show_collisions" type="bool" value="true" />

    <param name="visualize_plan" type="bool" value="true" />

    <param name="refresh_interval_collision_map" type="double" value="0.0" />
    <param name="refresh_interval_kinematic_state" type="double" value="1.0" />
    <param name="bounding_planes" type="string" value="0 0 1 -0.01" />

    <param name="visualize_plan" type="bool" value="true"/>
    <param name="visualize_joint_goal" type="bool" value="true"/>

    <param name="wait_for_visualizer" type="bool" value="false"/>
    <param name="wait_for_visualizer_timeout" type="double" value="10.0"/>

  </node>
</launch>

